src/miniscript/mod.rs

```diff
@@ -34,8 +34,8 @@ use bitcoin_hashes::hash160;
 pub mod astelem;
 pub mod decode;
 pub mod lex;
-pub mod satisfy;
 pub mod types;
+pub mod satisfy;
```

Comment:
Was there a need for this reordering? `s` comes before `t`.

Reply:
No reason. I don't know why I did this 🤦‍♂